runtime_services/src/runtime_services.rs

```diff
@@ -16,10 +29,15 @@ use core::{
 };
 
 use r_efi::efi;
-pub use variable_services::{GetVariableStatus, VariableIdentifier, VariableInfo, VariableNameIterator};
-
-/// This is the runtime services used in the UEFI.
-/// it wraps an atomic ptr to [`efi::RuntimeServices`]
+use variable_services::{GetVariableStatus, VariableInfo};
+
+/// The UEFI spec runtime serivces.
+/// It wraps an [`AtomicPtr`] around [`efi::RuntimeServices`]
+///
+/// UEFI Spec Documentation:
+/// <a href="https://uefi.org/specs/UEFI/2.10/08_Services_Runtime_Services.html" target="_blank">

```

Comment:
Please use the typical markdown language

```markdown
[8. Services - RuntimeServices](https://uefi.org/specs/UEFI/2.10/08_Services_Runtime_Services.html)
```

Reply:
Updated, thanks for the tip!